packages/reactotron-app/App/Foundation/SubscriptionsHeader.js

```diff
@@ -71,6 +72,9 @@ class SubscriptionsHeader extends Component {
           </div>
           <div style={Styles.center}>
             <div style={Styles.title}>{TITLE}</div>
+            <div>
+              <ConnectionSelector />
```

Comment:
TODO: Build out filtering out the state based on connection.
TODO: Figure out the best way to show state trees when "ALL" is selected... is that even possible. WOULD YOU ACTUALLY WANT THAT YOU CRAZY?

Reply:
I think we pick 1 "read/write", and the others are "write-only".